finals-2024-01-16/code/5048-dijana-damcevska/src/app/components/band-edit/band-edit.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="edit-form-container" *ngIf="!!band">
```

Comment:
There is a mismatch between the formControls in the markup and the component code, so this form just does not work - it should be relatively simple fix.
+3 points for an almost functional edit component